### DISCUSSION TITLE
MPD Page settings

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#15] [Fixed Issues]
The Bit Perfect 'No Volume' and 'DSD over PCM' options do not retain their settings. In addition if 'No Volume' is selected then the Volume options (Crossfade, Normalization or Replay Gain) are also cleared when the 'No Volume' option resets to off/disabled.

Chris

**Comment by AndyFx12** [comment]
A workaround for No Volume issue: Select "Disabled" at mixer-control combo-box.
DSD over PCM > no problem here.

**Comment by rern** [comment]
Fix with today's update.

**Reply by ceejayemm** [reply]
Applied 20201224 and No Volume issue seems to be fixed. Thanks

Chris
